packages/components/src/LightBox/LightBox.css

```diff
@@ -1,5 +1,5 @@
 .wrapper:global(.ril__outer) {
-  font-family: "Source Sans Pro", Helvetica, Arial, sans-serif;
+  font-family: "Jobber Pro", "Source Sans Pro", Helvetica, Arial, sans-serif;
```

Comment:
We want to use Inter as our Source replacement - Jobber Pro is the in-Jobber Poppins replacement

Reply:
Oh woops, did I switch them around?

Reply:
Yep, swapped them back [here](https://github.com/GetJobber/atlantis/pull/952/commits/1c41b2eb62ba9573b9152baa8e45b5140cea612c) though so we're good